domains/pinkyplant.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Jay's website and portfolio.",
+    "repo": "https://github.com/pinkyplant/pinkyplant.github.io",
+    "owner": {
+        "username": "pinkyplant",
+        "email": "pinkyplantz@proton.me",
+        "twitter": "pinkyplant"
+    },
+    "record": {
+        "CNAME": "pinkyplant.github.io"
```

Comment:
Please add more stuff to your website (for example an about me)

Reply:
Okay thank you. Will resolve ASAP